docs/pages/status.js

```diff
@@ -43,7 +43,65 @@ export default function SearchFieldPage({ generatedDocGen }: {| generatedDocGen:
           />
         </MainSection.Subsection>
       </MainSection>
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use Status to communicate a step in a workflow or the state of an item."
+            defaultCode={`
+<Status type="ok" accessibilityLabel="This item is ok" />
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use SVGs or images that resemble the Status’ symbols to denote status."
+            defaultCode={`
+<Icon icon="workflow-status-problem" size="24" accessibilityLabel="This item has an error" />
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Place Status close to the status subject in question to provide context and reference. It can be placed as an inline element or paired side by side as needed.  "
+            defaultCode={`
+<Flex gap={1} alignItems="center">
+  <Status accessibilityLabel="This item is complete" type="ok" />
+  <Text weight="bold" size="300">Campaign complete</Text>
+</Flex>
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Place Status far away from its subject. "
+            defaultCode={`
+<Flex gap={12} direction="column" alignItems="center">
+  <Status accessibilityLabel="This item is paused" type="halted" />
+  <Text weight="bold" size="300">Campaign paused</Text>
+</Flex>
 
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use a title when the status it represents is unique, specific and critical for the user to know."
+            defaultCode={`
+<Status type="inProgress" title="Pending review" />
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use Status' sub text to display extraneous messaging."
```

Comment:
ditto here: `subtext` refers to the prop, so needs backticks and should match the prop name exactly

Reply:
Same comment for this one. I think there's solid arguments on both sides here and I don't know if I have an informed opinion of how much friction this would cause designers.

Reply:
Done!